server/middleWare/auth.js

```diff
@@ -0,0 +1,37 @@
+import jwt from 'jsonwebtoken'
+
+import { TEST_ACCESS_TOKEN } from '../tests/constants'
+
+const getToken = (cookie) => {
```

Comment:
@AStashevskaya Could we use `getCookie` from `client/utils/getCookie.js` instead to avoid code repeating? You can add second parameter which checks if `document` is defined and use `document.cookie` by default